install/streams.csv

```diff
@@ -2387,8 +2407,9 @@ WHSB,2F,,,50,true,false,false,2009-04-15T00:00:04Z,2019-12-16T00:00:00Z
 WHSB,2F,,,200,true,false,false,2009-04-15T00:00:04Z,9999-01-01T00:00:00Z
 WHSB,2G,,,50,true,false,false,2009-04-15T00:00:04Z,2019-12-16T00:00:00Z
 WHSB,2G,,,200,true,false,false,2009-04-15T00:00:04Z,9999-01-01T00:00:00Z
-WHSZ,10,L,H,1,false,false,false,2020-06-19T00:00:00Z,9999-01-01T00:00:00Z
-WHSZ,10,E,H,100,false,false,false,2020-06-19T00:00:00Z,9999-01-01T00:00:00Z
+WHSZ,10,L,H,1,false,false,false,2021-11-03T22:57:00Z,9999-01-01T00:00:00Z
+WHSZ,10,E,H,100,false,false,false,2020-06-19T00:00:00Z,2021-11-03T02:58:00Z
+WHSZ,10,H,H,100,false,false,false,2021-11-03T22:57:00Z,9999-01-01T00:00:00Z
```

Comment:
This looks fine, with the history of sensor swaps:
(also just a note for me so I don't need to remind myself later why I have this comment 😄 )
- EH*  at LC 10 starts in 2020 with Lennartz and closes in 2021
- HH* at LC 10 starts in 2021 & still open, with Trillium 

However, I was confused why there are two location codes (10 and 11) at the same station coordinates with both broadband streams. The difference is LC 10 has non-axial components, whereas LC 11 has axial components. Does that sound confusing or I'm just confusing myself? 😄

Reply:
This is Whataroa River borehole, one sensor was at the surface and one down the hole, the one down the hole (11) would have been axial.